hpat/tests/test_series.py

```diff
@@ -73,6 +73,15 @@
 ]
 
 
+def gen_srand_array(size, nchars=8):
+    """Generate array of strings of specified size based on [a-zA-Z] + [0-9]"""
+    accepted_chars = list(string.ascii_letters + string.digits)
+    rands_chars = np.array(accepted_chars, dtype=(np.str_, 1))
+
+    np.random.seed(100)
+    return np.random.choice(rands_chars, size=nchars * size).view((np.str_, nchars))
```

Comment:
I think this is a bad idea to use random input data. If some test failed by some input data - it would be more difficult to reproduce the issue.

Reply:
BTW I set up seed here `np.random.seed(100)`. So the result of the function should be always identical. Anyway I will think how to avoid random generation.